pdf/model/appearance.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is subject to the terms and conditions defined in
+ * file 'LICENSE.md', which is part of this source code package.
+ */
+
+package model
+
+import "github.com/unidoc/unidoc/pdf/core"
+
+// PdfSignatureAppearance defines a signature with a specified form field and
+// annotation widget for appearance styling.
+type PdfSignatureAppearance struct {
```

Comment:
@adrg Can you check whether you feel that this type is a duplicate of PdfFieldSignature in fields.go?
https://github.com/unidoc/unidoc/blob/2ef5b5ab535600c50b8ad345bf38cd2783b41367/pdf/model/fields.go#L434
Technically this one is more high level for creating signatures, so I suppose it's OK, although it could be confusing.

Reply:
@gunnsth I do find it very similar to PdfFieldSignature. The only difference I can see is that PdfSignatureAppearance is also a widget. If that's the case, the PdfSignatureAppearance struct could be merged into PdfFieldSignature.

Can you please review my branch v3-a5i-pdf-sign. I merged the two structures there.
https://github.com/adrg/unidoc/tree/v3-a5i-pdf-sign